socrates/lib/site/views/history.jade

```diff
@@ -82,14 +84,22 @@ block content
         In 2013, the limited amount of beds became a problem. We discussed again what to do about it, and asked the organizers to seek a
         location that may scale up to 200 participants.
       h3 2014 #{''}
-        small(style='color:grey') Moving on
+        small(style='color:grey') Forth year
       p.
-        Rückersbach had an advantage: it was close to Frankfurt airport (about a one hour ride by car). That made it easy for people
-        from other countries to attend, since Frankfurt is the largest airport in Germany. It would be hard to find such a spot with
-        more beds in such a good position.
+        This year we finally switched to a new location. The choice fell on
```

Comment:
Ich würde das "finally" streichen - vielleicht ziehen wir ja nochmal in eine andere Location...